corehq/apps/app_manager/static/app_manager/js/apps_base.js

```diff
@@ -19,4 +19,16 @@ $(function () {
     if (v2) {
         hqLayout.utils.setIsAppbuilderResizing(true);
     }
+
+    $(document).ajaxComplete(function(e, xhr, options) {
+        if (/edit_form_attr/.test(options.url) ||
```

Comment:
not quite following - what's the case where false positives would happen? i was thinking of the version-checking because it's what we do on the releases page to verify that you can/can't make a new build.